packages/react-examples/src/react/FocusTrapZone/e2e/FocusTrapZone.e2e.ts

```diff
@@ -1,24 +1,495 @@
+import type { IFocusTrapZoneProps } from '@fluentui/react/lib/FocusTrapZone';
+import { FTZTestWindow } from './shared';
+
 const ftzStoriesTitle = 'Components/FocusTrapZone/e2e';
 
+/**
+ * Calls `window.setProps()` -- this must be defined by the story being tested
+ */
+function setProps(props: IFocusTrapZoneProps) {
+  // Use should() to ensure that the call retries if setProps isn't defined yet
+  // (this can happen in headless mode, if the example isn't finished rendering)
+  cy.window().should(win => {
+    (win as FTZTestWindow).setProps!(props);
```

Comment:
Setting this on window is OK in tests.  Most tests have an afterAll that sets it back to undefined to clean it up.